src/common/utils/kaltura-params.js

```diff
@@ -93,6 +93,15 @@ function getReferrer(): string {
   return referrer;
 }
 
+/**
+ * @return {string} - The referrer after URIComponent encoded
+ * @private
+ */
+function getEncodedReferrer(): string {
+  const referrer = getReferrer();
+  return typeof referrer === 'string' ? encodeURIComponent(referrer) : referrer;
```

Comment:
do we `typeof` this because of flow?

Reply:
when can it be undefined or not a string?

Reply:
in reality according to the current implementation of getReferrer it doesn't seem possible. Still i think it is good to be sure and wrap it in case getReferrer returns undefined in some weird scenario (for example an old browser or something)

Reply:
not sure I follow - do you know old browsers return undefined? according to MDN it is always a string: https://developer.mozilla.org/en-US/docs/Web/API/Document/referrer

Reply:
removed check